python/pyspark/sql/dataframe.py

```diff
@@ -267,11 +267,13 @@ def isStreaming(self):
         return self._jdf.isStreaming()
 
     @since(1.3)
-    def show(self, n=20, truncate=True):
+    def show(self, n=20, truncate=True, truncateTo=20):
```

Comment:
Yes, that will be cool, just little extra check for type of the variable. I was wondering if by default, I should leave it as `Boolean`. Not altering the API.